yjit.c

```diff
@@ -120,6 +120,55 @@ rb_yarv_insn_len(VALUE insn)
     return insn_len(insn);
 }
 
+unsigned int
+rb_yarv_vm_ci_argc(const struct rb_callinfo *ci) {
+    return vm_ci_argc(ci);
+}
+
+ID
+rb_yarv_vm_ci_mid(const struct rb_callinfo *ci) {
+    return vm_ci_mid(ci);
+}
+
+unsigned int
+rb_yarv_vm_ci_flag(const struct rb_callinfo *ci) {
+    return vm_ci_flag(ci);
+}
+
+rb_method_visibility_t
+rb_yarv_METHOD_ENTRY_VISI(rb_callable_method_entry_t *me) {
+    return METHOD_ENTRY_VISI(me);
+}
+
+bool
+rb_yarv_RB_TYPE_P(VALUE obj, enum ruby_value_type t)
+{
+    return RB_TYPE_P(obj, t);
+}
+
+VALUE
+cme_get_defined_class(rb_callable_method_entry_t* cme) {
+    return cme->defined_class;
+}
+
+rb_method_type_t
+cme_get_def_type(rb_callable_method_entry_t* cme)
+{
+    return cme->def->type;
+}
+
+ID
+cme_get_def_body_attr_id(rb_callable_method_entry_t* cme)
+{
+    return cme->def->body.attr.id;
+}
+
+enum method_optimized_type
+cme_get_def_body_optimized_type(rb_callable_method_entry_t* cme)
+{
+    return cme->def->body.optimized.type;
+}
+
```

Comment:
Okay. So for the cme structure, fields "def->type", you'd prefer "get_cme_def_type" to "cme_get_def_type"? Fair enough. These are all going to be a bit word-salad anyway -- the "right" way to do it would have far, far too many definitions.

But yeah, consistency is good.

Reply:
Yeah I'd prefer `rb_get_cme_def_type`. I agree that this is pretty word salad. That' part of why we're trying to move to Rust 😅